storage/elasticsearch/client.go

```diff
@@ -47,24 +47,42 @@ const indexVersion = 11
 const indexPrefix = "skydive"
 const indexAllAlias = "all"
 
-// ElasticLimits describes index limits driving roll policy
-type ElasticLimits struct {
+// Config describes configuration for elasticsearch
+type Config struct {
+	ElasticHost  string
+	MaxConns     int
+	RetrySeconds int
+	BulkMaxDocs  int
+	BulkMaxDelay int
 	EntriesLimit int
 	AgeLimit     int
 	IndicesLimit int
 }
 
-// NewElasticLimitsFromConfig create new limits from configuration
-func NewElasticLimitsFromConfig(path string) ElasticLimits {
-	limits := ElasticLimits{}
-	limits.EntriesLimit = config.GetInt(path + ".index_entries_limit")
-	limits.AgeLimit = 0
+func NewConfig(name string) Config {
+	cfg := Config{}
+
+	second := "storage.elasticsearch"
+	first := second
+	if name != "" {
+		first += "." + name
+	}
+
+	cfg.ElasticHost = config.GetString(first+".host", second+".host")
```

Comment:
as everything is almost constant and static why not writing

se := "storage.elasticsearch"
cfg.ElasticHost = config.GetString(se + ".host", se+"."+name+".host" )

Reply:
this will break as `name` may not be "" and in that case what you proposed will evaluate to: 
```
GetString("storage.elasticsearch.host", "storage.elasticseach..host")
```